py/selenium/webdriver/firefox/webdriver.py

```diff
@@ -114,3 +116,25 @@ def firefox_profile(self):
 
     def set_context(self, context):
         self.execute("SET_CONTEXT", {"context": context})
+
+    @contextmanager
+    def using_context(self, context):
+        """Sets the context that Selenium commands are running in using
+        a `with` statement. The state of the context on the server is
+        saved before entering the block, and restored upon exiting it.
+
+        :param context: Context, may be one of the class properties
+            `CONTEXT_CHROME` or `CONTEXT_CONTENT`.
+
+        Usage example::
+
+            with self.selenium.using_context(self.selenium.CONTEXT_CHROME):
```

Comment:
Remove `self.` as it's not needed for the example. This example will also fail because `selenium` has no property named `CONTEXT_CHROME`.